vowpalwabbit/core/include/vw/core/gen_cs_example.h

```diff
@@ -93,7 +96,7 @@ void gen_cs_example_dm(cb_to_cs& c, VW::example& ec, const VW::cb_label& ld, VW:
       VW::cs_class wc = {0., cl.action, 0., 0.};
 
       // get cost prediction for this action
-      wc.x = CB_ALGS::get_cost_pred<is_learn>(c.scorer, c.known_cost, ec, cl.action, 0);
+      wc.x = VW::get_cost_pred<is_learn>(c.scorer, c.known_cost, ec, cl.action, 0);
```

Comment:
is this worthy of being in VW:: maybe a cb related seemed more fitting

Reply:
Discussed offline, punting on taking on more fine grained namespaces